requirements.txt

```diff
@@ -3,4 +3,5 @@ Django==4.1.10
 djangorestframework==3.13.1
 pytz==2022.1
 sqlparse==0.4.4
+mysql-connector-python==8.2.0
 # mysqlclient==1.4.3
```

Comment:
Fix EOF

Reply:
Done, thank you. But this must be fixed on the `main` branch, along with some grammar and stylistic mistakes in the "README.md" file 😉